.github/workflows/go.yaml

```diff
@@ -0,0 +1,51 @@
+name: GoLang Tests
+
+on:
+  schedule:
+    - cron: 0 */6 * * *
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: public-amd64-small
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go Action
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.5'
+      - name: Install dependencies
+        run: |
+          go install github.com/jstemmer/go-junit-report/v2@latest
+          go install gotest.tools/gotestsum@latest
+
+      - name: Run go-junit-report tests
+        uses: trunk-io/analytics-uploader@main
+        with:
+          # Path to your test results.
+          junit-paths: "**/*_test.xml"
+          # Provide your Trunk organization slug.
+          org-slug: trunk-staging-org
+          # Provide your Trunk API token as a GitHub secret.
+          token: ${{ secrets.TRUNK_STAGING_ORG_API_TOKEN }}
+          run: sh go/go-junit-report/runtests.sh
```

Comment:
I think this helper runtest is doing very little for us here. 

I think you can leave the readme.md files in place and just fix up the command lines to use the tools directly.

then the awkward `cd` commands can be removed

Reply:
I tried doing that originally. Unfortunately the `go` commands kept getting run in the wrong directory.  Doing `run: |` with multiple commands as an argument to our uploader action didn't work either. It seemed to only run the first of the multiple commands. So I used the shell as a temp fix until we figure out a better solution.

Reply:
Did you file a bug that that run: is not behaving like the run does in GH action step - we might need to fix that